### PR TITLE
Add pt_BR .po file

### PR DESCRIPTION
**Pull request by feikname**
This commit adds pt-BR translation (Brazilian Portuguese). 

I am am not sure if I should have executed the `update-pot.sh` script so this commit doesn't include the output from it. When I tried to run it, the last line said `../../Scripts/Main.as: not found`, is this the correct behaviour?

### CODE REVIEW

File: Resources/Locales/pt_BR/pt_BR.po [hunk 1]
```diff
@@ -0,0 +1,1773 @@
...
+#: ../../Scripts/Gui/Preferences.as:671
+msgctxt "Preferences"
+msgid "Feedbacks"
+msgstr "Feedbacks"
+
+#: ../../Scripts/Gui/Preferences.as:643 ../../Scripts/Gui/Preferences.as:671
+#: ../../Scripts/Gui/Preferences.as:672
+msgctxt "Preferences"
+msgid "Chat Notify Sounds"
+msgstr "Sosn de Notificação do Chat"
```
Comment (feikname) [inline Resources/Locales/pt_BR/pt_BR.po]: Typo here.

File: Resources/Locales/pt_BR/pt_BR.po [hunk 2]
```diff
@@ -0,0 +1,1773 @@
...
+
+#: ../../../Sources/Client/LimboView.cpp:184
+msgctxt "Client"
+msgid "Select Team:"
+msgstr "Selecione a Equipe:"
+
+#: ../../../Sources/Client/LimboView.cpp:191
+msgctxt "Client"
+msgid "Select Weapon:"
+msgstr "Seleciona a Arma:"
```
Comment (feikname) [inline Resources/Locales/pt_BR/pt_BR.po]: Typo. Again.

File: Resources/Locales/pt_BR/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,1773 @@
...
+#: ../../Scripts/Gui/StartupScreen.as:1103
+msgctxt "StartupScreen"
+msgid "The Sun causes lens flare."
+msgstr "O sol causa lens flare."
+
+#: ../../Scripts/Gui/StartupScreen.as:1095
+#: ../../Scripts/Gui/StartupScreen.as:1105
+msgctxt "StartupScreen"
+msgid "Flares for Dynamic Lights"
+msgstr "Flares para L0uzes Dinâmicas"
```
Comment (feikname) [inline Resources/Locales/pt_BR/pt_BR.po]: Another typo.

File: Resources/Locales/pt_BR/pt_BR.po [hunk 4]
```diff
@@ -0,0 +1,1773 @@
...
+#: ../../Scripts/Gui/StartupScreen.as:1111
+msgctxt "StartupScreen"
+msgid "Depth of Field"
+msgstr "Profundidade do Campo de Visão"
+
+#: ../../Scripts/Gui/StartupScreen.as:1102
+#: ../../Scripts/Gui/StartupScreen.as:1112
+msgctxt "StartupScreen"
+msgid "Blurs out-of-focus objects."
+msgstr "Desfoca objetos fora de foco."
```
Comment (feikname) [inline Resources/Locales/pt_BR/pt_BR.po]: Could be better.

File: Resources/Locales/pt_BR/pt_BR.po [hunk 5]
```diff
@@ -0,0 +1,1773 @@
...
+
+#: ../../Scripts/Gui/StartupScreen.as:1105
+#: ../../Scripts/Gui/StartupScreen.as:1143
+#: ../../Scripts/Gui/StartupScreen.as:1165
+#: ../../Scripts/Gui/StartupScreen.as:1115
+#: ../../Scripts/Gui/StartupScreen.as:1153
+#: ../../Scripts/Gui/StartupScreen.as:1175
+msgctxt "StartupScreen"
+msgid "Medium"
+msgstr "Médio"
```
Comment (feikname) [inline Resources/Locales/pt_BR/pt_BR.po]: Another error.

**Comment by yvt** [comment]
Cool!

> I am am not sure if I should have executed the `update-pot.sh` script so this commit doesn't include the output from it. When I tried to run it, the last line said `../../Scripts/Main.as`: not found, is this the correct behaviour?

I didn't get that message on my computer. I guess it's okay...

**Pull request merged**